### PR TITLE
platform/gce: wait for image creation to finish on request

### PR DESCRIPTION
**Pull request by pbx0**
When scripting image uploads before a kola run, ongoing image creation
operations would cause kola to fail. This change makes image uploads
wait for the image create operation to complete before returning.

### CODE REVIEW

**Comment by mischief** [comment]
lgtm

File: platform/gce.go [hunk 1]
```diff
@@ -430,6 +440,35 @@ OpLoop:
...
+		if err != nil {
+			return fmt.Errorf("Failed to get op %s: %v", opname, err)
+		}
+		switch op.Status {
+		case "PENDING", "RUNNING":
+			continue
+		case "DONE":
+			if op.Error != nil {
+				for _, operr := range op.Error.Errors {
+					return fmt.Errorf("Error creating instance: %+v", operr)
```
Comment (vcaputo) [inline platform/gce.go]: drive-by observation: we don't typically capitalize returned error strings

Comment (pbx0) [inline platform/gce.go]: gah, just merged it. Noted though.

Comment (mischief) [inline platform/gce.go]: :+1: but we can worry about it later.
